tests/Piral.Blazor.Tests.sln

```diff
@@ -13,6 +13,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Piral.Blazor.SmokeTests.Pil
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Piral.Blazor.SmokeTests.PiletB", "Piral.Blazor.SmokeTests.PiletB\Piral.Blazor.SmokeTests.PiletB.csproj", "{EE84FF14-E742-475A-AA87-C800CFEDDDFF}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Piral.Blazor.Utils", "..\src\Piral.Blazor.Utils\Piral.Blazor.Utils.csproj", "{003F947A-B6A2-49AF-BCF9-05697CC1E030}"
```

Comment:
Any reason for adding this?

Reply:
Piral.Blazor.SmokeTests got a reference to Piral.Blazor.Utils, but it is outside of the solution scope.
In a clean checkout Piral.Blazor.Utils will not be builded and tests can't be run.

Reply:
Well, yes, build comes before test - as the CI/CD pipeline does (which also runs the tests).

We can keep it but honestly I'd prefer if tests is actually just that.